docs/book/middleware.md

```diff
@@ -18,6 +18,17 @@ is callable. When those conditions are met, it uses the [PSR-7 bridge](https://g
 to convert the laminas-http request and response objects into PSR-7 instances, and
 then invokes the middleware.
 
+## Deprecation
+
+Starting with version 3.2.0 `Laminas\Mvc\MiddlewareListener` is deprecated.
+Middleware support is now available via
+[laminas-mvc-middleware](https://github.com/laminas/laminas-mvc-middleware)
+package, which acts as a drop-in replacement at `^1.0` or provides a PSR-15
+Middleware compatibility starting with `^2.0`.
+
+`laminas-mvc-middleware` is an MVC module that needs to be registered with
+laminas-mvc application to function.
```

Comment:
We can added a page with instructions for installations in the documentation of laminas-mvc-middleware which covers the module registration.

An example can be found in the documentation of laminas-i18n:

* Page in docs: https://docs.laminas.dev/laminas-i18n/installation/
* Configuration: https://github.com/laminas/laminas-i18n/blob/327fdc2d07572e273f8122f46b52c7475a730826/mkdocs.yml#L48-L52

The link can then be added here.